tests/integrationtests/remotes/test_remotes.py

```diff
@@ -0,0 +1,76 @@
+from argparse import Namespace
+
+import conans.client.conan_api
+import pytest
+
+import os
+from src.mumoco import mumoco_api
+import src.conanbuilder.remote
+
+# Test 1
+# given
+# a clean conan setup
+# just conan center exist in remotes
+
+# when
+# running mumoco --remotes with config-build.json
+# then
+# has added the new remote ( test with conan remotes
+
+# Test 2 - test with empty remotes
+# Test 3 - test with multiple different remotes
+# Test 4 - test with multiple same remotes
+# Test 5 - test twice calling conan --remote and a changes config
+# ... what eslse?
+
+FILE_PATH = os.path.dirname(os.path.abspath(__file__))
+
+
+def test_empty_remote():
+    with pytest.raises(Warning):
+        mumoco_api(
+            remotes=True,
+            root=f"{FILE_PATH}/sourcetest_src_in_git",
+            config_file_path=f"{FILE_PATH}/empty_remote.json",
+        )
+
+
```

Comment:
@stefaneicher Apart from no remote test, I would not want to modify my conan remotes while running any test...